source/_layouts/guide.liquid

```diff
@@ -0,0 +1,481 @@
+---
+layout: has_no_sidebars
+title: Sass Basics
+introduction: >
+  Before you can use Sass, you need to set it up on your project. If you want to
+  just browse here, go ahead, but we recommend you go install Sass first. [Go
+  here](/install) if you want to learn how to get everything set up.
+---
+
+<section id="topic-1">
+  {% markdown %}## Preprocessing
```

Comment:
Our custom `{% markdown %}` tag should be equivalent to the built-in `{% renderTemplate %}` tag... For consistency, maybe we should remove ours, and just use e.g. `{% renderTemplate 'liquid,md' %}`?

Reply:
@jgerigmeyer would we have to replace each `markdown` tag with the `renderTemplate` one? Or could we use one `renderTemplate` tag at the top of the file and wrap all the contents in that tag? I'm testing it out locally and it looks like the we might have to do the first option which I feel less excited about 😆

Reply:
Yeah, probably the latter... But I could go either way on this. On the one hand it seems odd to add our own custom tag, which just duplicates built-in functionality. But on the other hand our tag is slightly less verbose, so 🤷

Reply:
@jgerigmeyer I got the `renderTemplate` tag working at the top of the file but then decided to keep the `markdown` tag after all because we started to get weird spacing issues again

Reply:
Oh, interesting! You got spacing issues replacing `{% markdown %}` and `{% endmarkdown %}` with `{% renderTemplate 'liquid,md' %}` and `{% endrenderTemplate %}` throughout? Or when trying to just use one `{% renderTemplate 'liquid,md' %}` at the top of the file?

The latter won't work, because then everything in the file will be parsed as Markdown -- which is the same problem we had before using a `.md` file. We need to explicitly be able to say when we want Markdown parsing to start and stop, which I think we can do with either `{% renderTemplate 'liquid,md' %}` or `{% markdown %}` equally?

Reply:
@jgerigmeyer yep, I was trying to see if I could get rid of all the various `markdown` tags with one `renderTemplate` tag at the top of the file but it brought back the spacing issues we were seeing with the `.md` file.

Reply:
Looking at both options, I like the `markdown` tag as opposed to `renderTemplate` because it reads a bit more clearly to me so I'm in favor of keeping ours ✨